.github/workflows/update_disabled_tests.yml

```diff
@@ -16,6 +16,18 @@ jobs:
           # score changes every request, so we strip it out to avoid creating a commit every time we query.
           curl 'https://api.github.com/search/issues?q=is%3Aissue+is%3Aopen+repo:pytorch/pytorch+in%3Atitle+DISABLED' \
           | sed 's/"score": [0-9\.]*/"score": 0.0/g' > disabled-tests.json
+      - name: Sort disabled-tests.json
+        shell: python
+        run: |
+          import json
+          with open('disabled_tests.json') as file:
+            disabled_tests_json = json.load(file)
+
+          # Sort the items (disabled tests) by their URL
+          disabled_tests_json['items'].sort(key=lambda x: x['url'])
```

Comment:
Can we add a note of what url might look like in this context?

Should make it easier for anyone else coming afterwards to know what it’s getting sorted on